### DISCUSSION TITLE
Issue with tuist with `v2.4.1`

### DISCUSSION THREAD

**Discussion by ladislas** [gonzalezreal/swift-markdown-ui#370] [Q&A] [answered]
Hi there! :) 

I'm working with [tuist](https://github.com/tuist/tuist) to setup my project. I've tried to upgrade swift-markdown-ui from `v2.4.0` to `v2.4.1` and I now have an error when building the project:

```
/Users/ladislas/dev/leka/ios-monorepo/Tuist/.build/checkouts/swift-cmark/extensions/tasklist.h
/Users/ladislas/dev/leka/ios-monorepo/Tuist/.build/checkouts/swift-cmark/extensions/tasklist.h:6:1 Missing '#include "cmark-gfm.h"'; 'cmark_syntax_extension' must be declared before it is used
```

I've tried adding it manually but it goes through the rabbit hole and I ended up editing a lot of files with the error moving to a new one each time.

Any ideas on how to fix this? :)

Thanks for your kind help.

**Comment by gonzalezreal** [comment, marked as answer]
Hi @ladislas,

There is an open issue for the `swift-cmark` dependency in the Tuist project:
tuist/tuist#6879

The workaround is to add the `swift-markdown-ui` as a regular SPM dependency in the `Project` initializer:

```swift
let project = Project(
    name: "MyApp",
    organizationName: "MyOrg",
    packages: [
        // workaround for https://github.com/tuist/tuist/issues/6879
        .remote(url: "https://github.com/gonzalezreal/swift-markdown-ui", requirement: .upToNextMajor(from: "2.4.1")),
    ],
    // ...
)
```

**Reply by ladislas** [reply]
ah thanks so much! I should have looked at the tuist issues before posting :) 

I'll wait for the definitive fix then as we have a lot of projects using it :)
